fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1279,6 +1279,38 @@ Future< Standalone<RangeResultRef> > ReadYourWritesTransaction::getRange(
 		}
 	}
 	
+	// Use special key prefix "\xff\xff/transaction/conflicting_keys/<some_key>",
+	// to retrieve keys which caused latest not_committed(conflicting with another transaction) error.
+	// The returned key value pairs are interpretted as :
+	// <key1> : '1' - any keys equal or larger than this key are (probably) conflicting keys
+	// <key2> : '0' - any keys equal or larger than this key are (definitely) not conflicting keys
+	// Due to the implementation of resolver, currently, 
+	// we can only give key ranges that contain at least one key is conflicting.
+	KeyRef conflictingKeysPreifx = LiteralStringRef("\xff\xff/transaction/conflicting_keys/");
+	// TODO : This condition need to be changed in the future when we have more special keys under "\xff\xff/transaction/"
+	if (begin.getKey().startsWith(conflictingKeysPreifx) && end.getKey().startsWith(conflictingKeysPreifx)) {
+		// Remove the special key prefix "\xff\xff/transaction/conflicting_keys/"
+		KeyRef beginConflictingKey = begin.getKey().removePrefix(conflictingKeysPreifx);
+		KeyRef endConflictingKey = end.getKey().removePrefix(conflictingKeysPreifx);
+
+		// Check if the conflicting key range to be read is valid
+		KeyRef maxKey = getMaxReadKey();
+		if(beginConflictingKey > maxKey || endConflictingKey > maxKey)
+			return key_outside_legal_range();
+		
+		begin.setKey(beginConflictingKey);
+		end.setKey(endConflictingKey);
+		if (tr.info.conflictingKeysRYW) {
+			Future<Standalone<RangeResultRef>> resultWithKeyPrefixFuture = 
+				tr.info.conflictingKeysRYW->getRange(begin, end, limits, snapshot, reverse);
+			// Make sure it happens locally
+			ASSERT(resultWithKeyPrefixFuture.isReady());
+			return resultWithKeyPrefixFuture.get();
```

Comment:
You have to add the prefix back to the keys here.

Reply:
Sorry for the bad naming. I forgot to change it.
The results are with no prefix right now.
The current behavior is when you call `getRagne("\xff\xff/transaction/conflicting_keys/<start_key>, ", "\xff\xff/transaction/conflicting_keys/<end_key>, ",)`
The keys returned are not with prefix like:
`far: 1; fool: 0`
I think it is enough for now since currently there is only conflicting keys right now.

Reply:
As we discussed, I have added the prefix to the returned keys, which will keep the consistency of `getRange`.